tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Good question.

Yes, the tsconfig.json is a normal part of a TypeScript project. If a project supports TypeScript, it will very likely have such a file. The additional file, declaration.tsconfig.json, extends upon it.

Reply:
Makes sense.